cmd/nitro/init.go

```diff
@@ -644,6 +644,10 @@ func openInitializeChainDb(ctx context.Context, stack *node.Node, config *NodeCo
 					return chainDb, l2BlockChain, err
 				}
 				if config.Init.RecreateMissingStateFrom > 0 {
+					err = gethexec.PopulateStylusTargetCache(&config.Execution.StylusTarget)
```

Comment:
Thank you for the report and solution!

Instead of adding a copy of stylus initialization here, please move the existing call to PopulateStylusTargetCache to be before openInitializeChainDb. That way we only need one call to PopulateStylusCacheTarget.